docs/src/SUMMARY.md

```diff
@@ -0,0 +1,10 @@
+# Summary
+
+- [Welcome to Robotnix](welcome.md)
```

Comment:
Uppercase R or not?

Elsewhere it seems like it's uppercased only when a word would otherwise be. And none of the other links use title case. So I assume there is no title case here.

Reply:
I may not have been very consistent, but yes, the default should be lowercase. In the context of headings or titles we can still use capital R.  Part of my motivation for not titling the project "RobotNix" was to retain the fun comparison with the word "robotnik", and I feel like using the lowercased "robotnix" instead of "Robotnix" helps a little to not overly emphasize the "robot" part of the name.

Reply:
Sorry, that was actually my fault.

Reply:
I've opted to use titlecase for these links. So, the capital "R" stays for now :)